### ISSUE TITLE
ERROR: Regular TensorFlow ops are not supported by this interpreter. Make sure you apply/link the Flex delegate before inference. ERROR: Node number 484 (FlexFusedBatchNormV3) failed to prepare.

### ISSUE THREAD

**Issue by cookingbear** [tensorflow/tensorflow#44543]
I use the version tf-nightly==2.5.0-dev20201029 and tested the tflite model

I converted the model into tflite and the python code below can run successfully. 
```
interpreter = tf.lite.Interpreter(model_path="model.tflite")
for samples in data_queue:
            input_details = interpreter.get_input_details()
            output_details = interpreter.get_output_details()
            interpreter.resize_tensor_input(input_details[0]['index'], samples["input"].shape)
            interpreter.allocate_tensors()
            interpreter.set_tensor(input_details[0]['index'], samples["input"])
            interpreter.invoke()
            features = interpreter.get_tensor(output_details[0]['index'])
            self.vocoder(features.numpy()) 
```

However, when I use C++ code to test the tflite model, the erros below showed:

ERROR: Regular TensorFlow ops are not supported by this interpreter. Make sure you apply/link the Flex delegate before inference.
ERROR: Node number 484 (FlexFusedBatchNormV3) failed to prepare.

**Comment by Saduf2019** [comment]
@cookingbear 
Please refer ot this issie with same error and let us know: #40157 , [link](https://github.com/tensorflow/tensorflow/issues/40677#issuecomment-653680600).

Else please share a simple standalone to reproduce the error? Thanks!

**Comment by cookingbear** [comment]
> @cookingbear
> Please refer ot this issie with same error and let us know: #40157 , [link](https://github.com/tensorflow/tensorflow/issues/40677#issuecomment-653680600).
> 
> Else please share a simple standalone to reproduce the error? Thanks!

the code can be seen in https://github.com/tensorflow/tensorflow/issues/44520
and I do not think the problem is same as that from the link you shared.
can you please look into this?

**Comment by Saduf2019** [comment]
@cookingbear 
The code shared in #$4520 does not have the error reported, i ran the code and see aonly a warning, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/88d4f6ce278ff55dc0bb404a4eb7f331/untitled458.ipynb).
Can you please share a colab gist of the error reported,

**Comment by cookingbear** [comment]
> @cookingbear
> The code shared in #$4520 does not have the error reported, i ran the code and see aonly a warning, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/88d4f6ce278ff55dc0bb404a4eb7f331/untitled458.ipynb).
> Can you please share a colab gist of the error reported,


I just found that if I delete the line below, the similar errors metioned above would show during the tflite model converting process. the other codes remained the same.

```
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
                                                                     tf.lite.OpsSet.SELECT_TF_OPS]
```

errors:

error: 'tf.FusedBatchNormV3' op is neither a custom op nor a flex op

<unknown>:0: error: failed while converting: 'main': Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag):
        tf.FusedBatchNormV3 {data_format = "NHWC", device = "", epsilon = 1.000000e-03 : f32, exponential_avg_factor = 1.000000e+00 : f32, is_training = true}


The errors above are also related with FusedBatchNormV3 which is the same as that showed by inference using c++

**Comment by cookingbear** [comment]
> > @cookingbear
> > The code shared in #$4520 does not have the error reported, i ran the code and see aonly a warning, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/88d4f6ce278ff55dc0bb404a4eb7f331/untitled458.ipynb).
> > Can you please share a colab gist of the error reported,
> 
> I just found that if I delete the line below, the similar errors metioned above would show during the tflite model converting process. the other codes remained the same.
> 
> ```
> converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
>                                                                      tf.lite.OpsSet.SELECT_TF_OPS]
> ```
> 
> errors:
> 
> error: 'tf.FusedBatchNormV3' op is neither a custom op nor a flex op
> 
> :0: error: failed while converting: 'main': Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag):
> tf.FusedBatchNormV3 {data_format = "NHWC", device = "", epsilon = 1.000000e-03 : f32, exponential_avg_factor = 1.000000e+00 : f32, is_training = true}
> 
> The errors above are also related with FusedBatchNormV3 which is the same as that showed by inference using c++

I mean that if you delete that line mentioned above in the colab, the errors would show. If this problem can be solved, maybe the original problem can be solved too.

**Comment by cookingbear** [comment]
I just found that the error is related with LayerNormalization. If I replace it with BatchNormalization, it can run successfully. But in my code, LayerNormalization is necessary. Although I can convert the code to the tflite model, it still show the error during inference using c++

**Comment by jvishnuvardhan** [comment]
@cookingbear Please check this [comment](https://github.com/tensorflow/tensorflow/issues/43934#issuecomment-714832778) which is similar to your issue. Also, check [this resource](https://www.tensorflow.org/lite/guide/ops_select) on how to enable `Select_TF_Ops`. Thanks!
